aiohttp/multipart.py

```diff
@@ -237,11 +237,16 @@ class BodyPartReader:
 
     chunk_size = 8192
 
-    def __init__(self, boundary: bytes,
-                 headers: Mapping[str, Optional[str]],
-                 content: StreamReader) -> None:
+    def __init__(
+        self,
+        boundary: bytes,
+        headers: Mapping[str, Optional[str]],
+        content: StreamReader,
+        newline: bytes = b'\r\n',
```

Comment:
To not cause any confusion, may be mark it somehow as internal? Like kwarg only or `_newline`. What do you think?

Reply:
Makes sense for both keyword only and `_newline` proposals.
It will prevent users from using this *special* argument.

Reply:
Ok, will make both changes.

Reply:
Fixed in https://github.com/aio-libs/aiohttp/pull/3492/commits/5536671a48e9f72a4ff57f13c7a6ea895bbbd9fd